src/test/java/com/wikia/webdriver/common/core/drivers/Browsers/GhostBrowser.java

```diff
@@ -0,0 +1,33 @@
+package com.wikia.webdriver.common.core.drivers.Browsers;
+
+import com.wikia.webdriver.common.core.WikiaWebDriver;
+import com.wikia.webdriver.common.core.drivers.BrowserAbstract;
+
+import org.openqa.selenium.phantomjs.PhantomJSDriver;
+import org.openqa.selenium.phantomjs.PhantomJSDriverService;
+import org.openqa.selenium.support.events.EventFiringWebDriver;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.openqa.selenium.support.events.EventFiringWebDriver'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)